examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -7700,13 +7700,13 @@ endpoint 0 {
   }
 
   server cluster LocalizationConfiguration {
-    persist  attribute activeLocale default = "en-US";
+    ram      attribute activeLocale default = "en-US";
```

Comment:
Is this so that the "default value" can be in the ZAP file and then endpoint config, and then the active locale is managed via storage at startup time, but also loaded from ember in the init?

Reply:
Yes.